CRM/Utils/Token.php

```diff
@@ -256,7 +256,7 @@ function ($matches) use (&$domain, $html, $escapeSmarty) {
    *
    * @return mixed|null|string
    */
-  public static function getDomainTokenReplacement($token, &$domain, $html = FALSE, $escapeSmarty = FALSE) {
+  public static function getDomainTokenReplacement($token, $domain, $html = FALSE, $escapeSmarty = FALSE) {
```

Comment:
$domain is not altered in this function, ergo it's not altered in the function that calls this function